standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/service/AcidHouseKeeperService.java

```diff
@@ -78,17 +86,13 @@ public void run() {
     }
   }
 
-  private void cleanTheHouse() {
+  void cleanTheHouse() {
     performTask(txnHandler::performTimeOuts, "Cleaning timed out txns and locks");
     performTask(txnHandler::performWriteSetGC, "Cleaning obsolete write set entries");
     performTask(txnHandler::cleanTxnToWriteIdTable, "Cleaning obsolete TXN_TO_WRITE_ID entries");
-    if (isCompactorEnabled) {
-      performTask(txnHandler::removeDuplicateCompletedTxnComponents, "Cleaning duplicate COMPLETED_TXN_COMPONENTS entries");
-      performTask(txnHandler::purgeCompactionHistory, "Cleaning obsolete compaction history entries");
-    }
   }
 
-  private void performTask(FailableRunnable<MetaException> task, String description) {
+  void performTask(FailableRunnable<MetaException> task, String description) {
```

Comment:
why have you removed private accessor?

Reply:
changing to protected, so that it can be accessed in sub class